dev/com.ibm.ws.jsp.2.3_fat/bnd.bnd

```diff
@@ -34,9 +34,12 @@ tested.features:\
     servlet-6.0, \
     expressionLanguage-5.0,\
     pages-3.1,\
-    cdi-4.0
-
-
+    cdi-4.0,\
+    expressionLanguage-6.0,\
```

Comment:
Can we fix this up to be consistent with the work we just did with lowercase in the tested.features directive?